app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+    required: "Yes",
+    api: {
+      param: "work_experience.jobs[].organisation_name",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["service"],
+      what: "uses this label to distinguish between different jobs when reviewing answers"
+    }, {
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s background"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["We don’t directly, but a reference from an employer may collaborate some of this information"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Job - Time commitment",
+  items: {
+    question: "Was this job full-time or part-time?",
+    required: "Yes",
+    api: {
+      param: "work_experience.jobs[].commitment",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s background"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["We don’t directly, but a reference from an employer may collaborate some of this information"],
+    notes: ["Feedback from HEIs has suggested hours per week is more useful (this is what UCAS asks)"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Job - Period of employment",
+  items: {
+    question: "Start date
+      End date",
+    required: "Yes",
+    api: {
+      param: "work_experience.jobs[].start_date
+      work_experience.jobs[].end_date",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["service"],
+      what: "uses this to calculate any gaps in work history"
+    }, {
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s background"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["We don’t directly, but a reference from an employer may collaborate some of this information"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Job - Skills and experience",
+  items: {
+    question: "Skills and experience relevant to teaching you gained in this role",
+    required: "Yes",
+    api: {
+      param: "work_experience.jobs[].description",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
+    },
+    why: [{
+      who: ["candidate"],
+      what: "may use this to demonstrate subject knowledge gained in the workplace"
+    }, {
+      who: ["provider"],
+      what: "uses this to build a picture of a candidate’s background"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["We don’t directly, but a reference from an employer may collaborate some of this information"]
```

Comment:
Notes – should we indicate why this field is different to a more standard text field for "job description"?